### PR TITLE
feat: change layout of ListingsCombined for mobile vs desktop

### PR DESCRIPTION
**Pull request by ifranch**
# Pull Request Template

## Issue Overview

This PR addresses #133 

- [ ] This change addresses the issue in full
- [x] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

- Separate CSS for ListingsCombined into its own file.
- Create different styles for desktop vs mobile. The desktop style is the existing style with 2 columns. The mobile style has the map in the background and the listings list in the foreground, covering only half the screen and with rounded corners.

Not included, will be in next PR:
- Expand/hide listings list on swipe.

## How Can This Be Tested/Reviewed?

Bring up the public site and go to the listings page.
Adjust the screen width/use the Chrome Device Emulator to see what the page looks like for desktop vs mobile.

Desktop:
image

Mobile:
image

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [x] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

**Review by isaac-looker — APPROVED** [review]

File: sites/public/src/components/listings/ListingsCombined.tsx [hunk 1]
```diff
@@ -1,28 +1,56 @@
...
+        <ListingsMap listings={props.listings} googleMapsApiKey={props.googleMapsApiKey} />
+      </div>
+      <div className={styles["listings-list"]}>
+        <ListingsList listings={props.listings}></ListingsList>
+      </div>
     </div>
-    <div style={{ overflowY: "auto", width: "600px" }}>
-      <ListingsList listings={props.listings}></ListingsList>
+  ) : (
+    <div className={styles["listings-combined__mobile"]}>
```
[the remark this answers is not in the document]
Comment (ifranch) [reply sites/public/src/components/listings/ListingsCombined.tsx]: Thanks for the pointer, that's more straightforward. Done

File: sites/public/src/components/listings/ListingsCombined.module.scss [hunk 2]
```diff
@@ -0,0 +1,45 @@
+.listings-combined {
+  // This is a not-ideal way to do "fill window minus header+footer" however I can't find another way to do this.
+  // TODO: update header+footer to a not-magic number
+  --listings-component-height: calc(100vh - 275px);
+  --listings-component-half-height: calc((100vh - 275px) / 2);
```
Comment (lisunshiny) [inline sites/public/src/components/listings/ListingsCombined.module.scss:5]: (nit) can this just be `calc(var(--listings-component-height) / 2)`?

**Review by lisunshiny — APPROVED** [review]
there is a bit of weird scrolling bc of the footer height, but we can address in a follow up (FYI I'm almost certain that there will be no footer in the hybrid mobile view, however, still waiting on what we do on desktop which is blocking the footer change on mobile as well)

File: sites/public/src/components/listings/ListingsCombined.module.scss (hunk 2 above)
Comment (ifranch) [reply sites/public/src/components/listings/ListingsCombined.module.scss]: Yep, done

**Pull request merged**